js/src/tooltip.js

```diff
@@ -70,6 +94,7 @@ const Tooltip = (() => {
     selector            : false,
     placement           : 'top',
     offset              : 0,
+    arrowPadding        : 6,
```

Comment:
Here you added a new config option, can you update `DefaultType` to handle that new option